homework-submission/react-hw4/src/components/stores.js

```diff
@@ -0,0 +1,39 @@
+import { observable, action, computed, configure } from "mobx";
+import Todos from "./todos.json";
+
+configure({ enforceActions: true });
+
+class Stores {
+  @observable
+  todos = Todos;
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+  };
+
+  @action
+  remover = item => {
+    const remove = this.todos.filter(elem => elem !== item);
+    this.todos = remove;
+  };
+
+  @action
+  submitHandler = e => {
+    e.preventDefault();
```

Comment:
Keep any event handling on the component, but in store just use naming the reflect the action that should be send to the backend, like `createTodo` instead of `submitHandler`